src/app/js/formats/title/TitleAdmin.js

```diff
@@ -21,7 +21,7 @@ const styles = {
 
 export const defaultArgs = {
     level: 1,
-    textColor: '#000',
+    textColor: '#818181',
```

Comment:
The idea is neat, but I would have used a constant (defined in an external file, perhaps within src/app/js/formats).